code/modules/projectiles/ammo_types/rocket_ammo.dm

```diff
@@ -232,8 +236,8 @@
 
 /datum/ammo/rocket/recoilless/heat/mech/on_hit_obj(obj/target_obj, obj/projectile/proj)
 	drop_nade(get_turf(target_obj))
-	if(ismecha(target_obj))
-		proj.damage *= 3 //this is specifically designed to hurt mechs
+	if(isvehicle(target_obj) || ishitbox(target_obj))
```

Comment:
Wondering if this should be a define.

Reply:
probably. Its annoying because in some situations you just want to hit either. In other situations you explicitly want to target the vehicle object itself.